app/src/androidTest/java/com/github/sdp/mediato/ProfileFragmentTest.java

```diff
@@ -72,9 +72,9 @@ public void setUp() {
 
   // Test whether the "Friends" button is displayed and contains the correct text

```

Comment:
I assume this is referring to the new following button. You can just correct the comment to avoid confusion in the future.

Reply:
I will change it next sprint during my refactorisation of the tests